src/linker/Linker/LinkContext.cs

```diff
@@ -435,6 +435,17 @@ public void LogMessage (MessageImportance importance, string message)
 			if (LogMessages && Logger != null)
 				Logger.LogMessage (importance, "{0}", message);
 		}
+
+		public void LogMessage (Message message)
+		{
+			if (Logger == null)
+				return;
+
+			if (message.Category == MessageCategory.Info && !LogMessages)
+				return;
```

Comment:
This is different from the existing `LogMessage (MessageImportance, string)` - that one will not log at all if `LogMessages` is false. If we're changing the behavior here I think it's worth a comment as to why.